### ISSUE TITLE
Management of Data and JavaScript Libraries while executing Data Driven Postman Collections

### ISSUE THREAD

**Issue by RJ05** [michaelruocco/gradle-postman-runner#3]
Hi,

Is there a way to specify location of data files used by Postman Collections or we may have to enhance to the whole solution to incorporate the same.

Thanks,
Rahul Jain

**Comment by simomat** [comment]
Are you talking about the collection specification files itself?

**Comment by RJ05** [comment]
Various other command line arguments which Newman provides https://www.npmjs.com/package/newman

`-g <source>, --globals <source>`
Specify file path or URL for global variables. Global variables are similar to environment variables but has a lower precedence and can be overridden by environment variables having same name.

`-d <source>, --iteration-data <source>`
Specify a data source file (CSV) to be used for iteration as a path to a file or as a URL. Read More

`-n <number>, --iteration-count <number>`
Specifies the number of times the collection has to be run when used in conjunction with iteration data file.

**Comment by RJ05** [comment]
PFA, Screenshot of execution some random characters on command prompt:-

![image](https://user-images.githubusercontent.com/11192088/28956460-6cef9c60-7930-11e7-96d7-793796235a15.png)

**Comment by simomat** [comment]
Good point, I didn't use newman for a while. These options must be new and should be integrated.

Regarding the strange chars in the terminal: I'm quite sure that's an issue with the terminal, that cannot interpret color codes or can't show unicode. Both can be disabled with the options.

**Comment by sakthiveltk** [comment]
I made required changes with new node plugins update and accept data file. Let me know, how I could contribute to commit the code and publish the plugins.

**Comment by simomat** [comment]
That's great. I'll just accepted the requests for now and trust it will work. I'm out for the weekend, so I can't check it by myself.
In general we have a lack of automatic testing the plugin. I was playing with the gradle plugin testing mechanism lately, but it's far from being ready...

**Comment by sakthiveltk** [comment]
Ok. Thats is not my pull request.  :). My pull request with title "Management of Data and JavaScript Libraries while executing Data Driven Postman Collections #3 "

**Comment by sakthiveltk** [comment]
Commit  done with my changes and tested with my repository. You can merge the changes from my forked repository or pull-request.

**Comment by sakthiveltk** [comment]
Hope you merge and pushing the changes to plugin repository.

**Comment by mventura912** [comment]
Are there plans to merge these changes and release a new version?

**Comment by ghost** [comment]
Can this pr be merged? Also, curious if csv file support can be added.

**Comment by ghost** [comment]
@simomat, can you please get this one merged.
